grpc.go

```diff
@@ -1449,3 +1449,26 @@ func (a *agentGRPC) SetGuestDateTime(ctx context.Context, req *pb.SetGuestDateTi
 	}
 	return &gpb.Empty{}, nil
 }
+
+func (a *agentGRPC) CopyFile(ctx context.Context, req *pb.CopyFileRequest) (*gpb.Empty, error) {
```

Comment:
We need unit tests for this - think security :)

Reply:
sure thing